packages/terra-popup/src/Popup.jsx

```diff
@@ -0,0 +1,370 @@
+import React from 'react';
```

Comment:
Potentially could be, I had considered it, but I don't really feel strong either way about it. Does anyone have a strong preference?